apps/nfdiv/nfdiv-cron-bulk-list-create/prod-00.yaml

```diff
@@ -1,27 +0,0 @@
-apiVersion: helm.toolkit.fluxcd.io/v2
-kind: HelmRelease
-metadata:
-  name: nfdiv-cron-bulk-list-create
-spec:
-  releaseName: nfdiv-cron-bulk-list-create
-  values:
-    job:
-      schedule: "0,30 5,6 * * *"
```

Comment:
I wonder why we had 2 config files for this cron with different timings - prod-00.yaml and prod/00.yaml? It looks like the job timing is 30 mins in the logs and I'm wondering whether this is the config we have been using in prod

Reply:
I have noticed that for a number of crons, the historical prod 00/01 yaml files have been retained even when they are not being referenced. That is why I was deleting those files. I will retain the historical files as of now. May be delete unused files as part of another task later on.